measures/Measures/test/tMeasJPL.cc

```diff
@@ -73,7 +73,10 @@ int main()
       os << "Mercury:    " << val << endl;
       MeasJPL::get(val, MeasJPL::DE200, MeasJPL::VENUS, dat);
       os << "Venus:      " << val << endl;
-      for (uInt i=0; i<3; i++) mvd1(i) = val(i); mvd1.adjust();
+      for (uInt i=0; i<3; i++) {
+        mvd1(i) = val(i);
+        mvd1.adjust();
```

Comment:
Shouldn't this be outside the loop?

Reply:
Yes, that's right, and actually that one test failed. I'll fix accordingly